app/service/rest.py

```diff
@@ -1322,3 +1328,38 @@ def get_service_join_request(request_id: uuid.UUID):
         raise InvalidRequest(message=f"Service join request with ID {request_id} not found.", status_code=404)
 
     return jsonify(service_join_request.serialize()), 200
+
+
+@service_blueprint.route("/update-service-join-request-status/<uuid:request_id>", methods=["POST"])
+def update_service_join_request(request_id: uuid.UUID):
+    data = request.get_json()
+
+    try:
```

Comment:
Minor thing, but we don't need to put error handling round `validate` because it throws a error which we already handle with an error handler. So we will already return a `400` with the messages.